docs/user/how-to/write-plans.rst

```diff
@@ -0,0 +1,203 @@
+Writing bluesky plans for blueapi
+=================================
+
+**Please read the following completely and carefully, as you risk losing data if plans are written incorrectly.**.
```

Comment:
See https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html
```suggestion
.. warning::
    Please read the following completely and carefully, as you risk losing data if plans are written incorrectly!.
```